files/common/config/tessera/Dockerfile

```diff
@@ -2,14 +2,14 @@ ARG TESSERA_VERSION=latest
 
 FROM quorumengineering/tessera:${TESSERA_VERSION}
 
-RUN apt-get update && apt-get install -y \
-  gettext \
-  && rm -rf /var/lib/apt/lists/*
+# develop uses a debain container, all releases use an alpine container - this allows both to be used for the quickstart
+# set the version in ../../.env
+RUN if [ -e /sbin/apk ] ; then apk add gettext --no-cache ; else apt-get update && apt-get install -y gettext && rm -rf /var/lib/apt/lists/* ; fi
```

Comment:
This should do alpine or debian base images and lets the user pick whichever variant they want